### PR TITLE
Assignee and Story were handled more abstractive.

### PR DESCRIPTION
**Pull request by hrsh7th**
- Assignee and Story were handled more abstractive.
- Cosmetic change: *List to *s
- Cosmetic change: JSDOc comment fixes

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -24,11 +24,16 @@ module.exports = {
     inline: true
   },
 
+  node: {
+    fs: 'empty'
+  },
```
Comment (hrsh7th) [inline webpack.config.js:29]: It's idiom on webpack.

File: src/component/task/RoadmapTask.js [hunk 2]
```diff
@@ -13,25 +15,24 @@ export class RoadmapTask {
...
     this.color = color;
     this.order = parseInt(order);
     this.from = new Date(from);
     this.to = new Date(new Date(to).setHours((new Date(to).getHours() + 24)));
-    this.involvement = parseInt(involvement);
-    if (this.involvement > 100) {
-      this.involvement = 100;
-    }
+    this.involvement = Math.min(parseInt(involvement), 100);
   }
```
Comment (hrsh7th) [inline src/component/task/RoadmapTask.js:35]: Simplify.

File: src/component/RoadmapCanvas.js [hunk 3]
```diff
@@ -34,35 +35,34 @@ export class RoadmapCanvas {
    * @param {Roadmap} roadmap
    */
   render(roadmap) {
-    if (this.type === "story") {
-      this._render(roadmap, roadmap.getSortedStoryList());
-    } else if (this.type === "assignee") {
-      this._render(roadmap, roadmap.getSortedAssigneeList());
+    // TODO: check. Is this condition need?
```
Comment (atago0129) [inline src/component/RoadmapCanvas.js:38]: Maybe **I** will not use `all` type (display  based on `sotry` and `assignee` ). So I think that it's ok to delete this condition.
But if **someone** wants to use `all` type, I can't delete this condition.
Or, refactor `Roadmap` and call `draw` on twice time.

Now
```javascript
let roadmap = new Roadmap.constructor(options, dataSeta);
roadmap.render();
```
After
```javascript
let roadmap = new Roadmap.constructor(dataSeta);
roadmap.render(options);
roadmap.render(options);
```

How do you think?

File: src/component/Roadmap.js [hunk 4]
```diff
@@ -11,106 +14,93 @@ export class Roadmap {
...
-    result.sort(function (a, b) {
-      if (a.order !== b.order) {
-        return a.order > b.order ? 1 : -1;
-      } else {
-        return a.name > b.name ? 1 : -1;
-      }
-    });
-    return result;
+  getTaskById(taskId) {
+    return this.tasks.filter(function(task) {
```
Comment (atago0129) [inline src/component/Roadmap.js:35]: 👍

File: src/component/task/RoadmapTask.js (hunk 2 above)
Comment (atago0129) [reply src/component/task/RoadmapTask.js:35]: +1

File: src/component/RoadmapCanvas.js (hunk 3 above)
Comment (hrsh7th) [reply src/component/RoadmapCanvas.js:38]: I think, Roadmappy should not supports this feature natively.

Roadmappy user has other way to solve this problem.

```js
let roadmap1 = new Roadmappy({ ..., type: 'story' });
roadmap1.render();
let roadmap2 = new Roadmappy({ ..., type: 'assignee' });
roadmap2.render();
```

Comment (hrsh7th) [reply src/component/RoadmapCanvas.js:38]: But if you need to support this feature, I think We should implement `interactive roadmap type change`.

Comment (atago0129) [reply src/component/RoadmapCanvas.js:38]: Oh, yes.
It's seems like a good idea.
We would like to implement on someday : )

**Comment by atago0129** [comment]
LGTM
then ma-shima

**Pull request merged**